neural_speed/models/llama/llama_utils.cpp

```diff
@@ -176,10 +192,24 @@ void Llama::load(model_context* ctx, model_progress_callback progress_callback,
       layer.norm[1] = ml->get_tensor(layers_i + ".ffn_norm.weight", {n_embd}, backend);
 
       // ffn GEMM
-      layer.ffn[0] = ml->get_tensor(layers_i + ".feed_forward.w1.weight", {n_embd, n_ff}, backend);
-      layer.ffn[1] = ml->get_tensor(layers_i + ".feed_forward.w2.weight", {n_ff, n_embd}, backend);
-      layer.ffn[2] = ml->get_tensor(layers_i + ".feed_forward.w3.weight", {n_embd, n_ff}, backend);
 
+      if (ml->verify_tensor(layers_i + ".feed_forward.w1.weight")) {
+        layer.ffn[0] = ml->get_tensor(layers_i + ".feed_forward.w1.weight", {n_embd, n_ff}, backend);
```

Comment:
Assert n_expert  and n_expert_used  to be zero here?

Reply:
OK

Reply:
done